backend-go/test/integration/settings_test.go

```
@@ -0,0 +1,71 @@
+	"github.com/yorkie-team/codepair/backend/test/helper"
+)
+
+func TestGetSettings(t *testing.T) {
+	conf := helper.NewTestConfig(t.Name())
+	codePair := helper.SetupTestServer(t)
+	_, access, _ := helper.LoginUserTestGithub(t, t.Name(), codePair.ServerAddr())
+
+	url := codePair.ServerAddr() + "/settings"
+
+	t.Run("get settings without file upload and yorkie intelligence", func(t *testing.T) {
+		conf.Storage.Provider = "false"
```

golangci-lint (1.64.8)

string `false` has 4 occurrences, make it a constant

(goconst)